CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+    - `bug`
+    - `feature`
+    - `documentation`
+    - `gh action`
+
+    For bugs and features include a label for the area of the project that is affected. Examples of appropriate labels are:
+        - `newsletter`
+        - `website`
+        - `podcast`
+
+3. Specify that you will work on this issue. This lets others know that someone is actively working on the project.
+4. Fork the repository on your profile and make changes locally. 
+
+#### If Wanting to Work on an Existing Issue
+
+**Claim** the issue by commenting on it or stating you will work on the issue. We will accept PRs from the first person (collaborating on an issue is encouraged).
+
+![Create a Branch from an Issue](.github/assets/Create%20Branch.jpg)
+
+### Automation with Github via GH Actions
+
+GH Actions allow you to automate the deployment of your code to GitHub.
+
+We intend to use GitHub Actions to automate the compilation of issues to consider for the show and the creation of at show pull request.
+
+We'll also use GitHub Actions to interact with issues as they relate to the associated [Project](https://github.com/users/kjaymiller/projects/4) boards.
+
+Just adding a missing period here.
+
+We'll also be using GH Actions to automate sending supporting content to our newsletter service (currently [ButtonDown](https://buttondown.email/)), and perhaps other services as we see fit. 
+
+You should submit a PR for a GitHub Actions if you believe the action will improve how we interact with our existing workflow.
+
+If you would like to pose a change to the existing workflow, please file and issue and wait for aggreement from the maintainers before working on the project.
+
+## General Administration around GitHub
+We use GitHub issues to communicate many changes that we would like to deploy to our website/podcast/newsletter and the inner workings of between them all. These may result in an issue with an `admin` label. 
+
+These are often changes to issue/PR templates or documentation that communicates the intention of the project.
+
+You are more than invited to work on these issues. Following the guidance given [above](#for-all-types-of-contributions).
+
+### Deployting to our third-party platforms via API
+
+We have to authenticate with our third-party platforms to deploy content to them and this requires an API key and often subject to rate-limiting. 
+
+If you are working on code that would connect to a Third Party, YOU MUST USE A TEST-CASE and mock the API call request and responses.
+
+We use [`httpx`](https://www.python-httpx.org) to make REST API calls to our third-party platforms and [`pytest-httpx`](https://pypi.org/project/pytest-httpx/) to mock those API calls in testing.
+
+We are not taking requests to change any of the services. Solicitation of services is not allowed and will be seen as Harassment and Handled according to our [CODE of CONDUCT](CODE_OF_CONDUCT.md).
```

Comment:
Can we remove this section until we've discussed #67?

Reply:
I have removed the paragraph.

Reply:
Thanks!